github-changelog-generator.php

```diff
@@ -1,242 +1,113 @@
 <?php
 
+// "autoload" our classes
+if (!class_exists('ins0\GitHub\Repository')) {
+    require_once 'src/Repository.php';
```

Comment:
I agree classes should not contain code like this. I can add these files to Composer's autoloading section for people who need the backwards compat and are using Composer, and if the user isn't using Composer then they can manually require themselves.